papermill/execute.py

```diff
@@ -2,6 +2,7 @@
 
 import copy
 import nbformat
+from pathlib import Path
```

Comment:
I just double checked since I wasn't sure -- `pathlib` was introduced in 3.4 and we require 3.6+ now. 👍